zcash_client_sqlite/src/wallet/commitment_tree.rs

```diff
@@ -775,17 +775,49 @@ pub(crate) fn put_shard_roots<
     // We treat the cap as a DEPTH-SHARD_HEIGHT tree so that we can make a batch insertion of
     // root data using `Position::from(start_index)` as the starting position and treating the
     // roots as level-0 leaves.
```

Comment:
```suggestion
    // We treat the cap as a tree with `DEPTH - SHARD_HEIGHT` levels, so that we can make a
    // batch insertion of root data using `Position::from(start_index)` as the starting position
    // and treating the roots as level-0 leaves.
```